package.json

```diff
@@ -4,19 +4,20 @@
 		"test:unit-test": "jest",
 		"test:end-to-end": "node tests/tester.js",
 		"test": "npm run test:end-to-end && npm run test:unit-test",
-
 		"watch": "onchange packages/*/src/ -- lerna run build",
-    
 		"purge": "rimraf packages/*/node_modules | rimraf packages/*/dist | rimraf packages/*/lib | rimraf node_modules",
-
 		"build": "lerna bootstrap && lerna run build"
 	},
 	"engines": {
 		"node": "~10",
 		"npm": "~6",
 		"yarn": "~1"
 	},
-	"dependencies": {},
+	"dependencies": {
```

Comment:
The root package.json of a monorepo with lerna does not have dependencies. This needs to be cleaned again

Reply:
👍